### PR TITLE
Update l10n.json | #1702

### PR DESCRIPTION
**Pull request by redmangospros**
#1702 
This AddsA A SemiCOlon Has Requested by @me-win in Issue #1702. This Adds The SemiCOlon Where Requested.

### CODE REVIEW

**Comment by redmangospros** [comment]
[Build Status](https://travis-ci.org/LLK/scratch-www)

**Comment by mewtaylor** [comment]
Thanks @redmangospros! Could you please include a bit more info in the PR description about what exactly this change does (i.e. what are the fixes)? It'll just be helpful when we take a look at the code and for testing.

**Comment by redmangospros** [comment]
I Think We're Okay?

**Comment by redmangospros** [comment]
Can Someone Help me with the travis fail? @GreenBayRules I Got this part, you can do The Img And Rest.

File: src/views/conference/2018/index/l10n.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "conference-2018.sessionItem4Desc": "Share what you've been doing in a short, lively presentation.",
 
     "conference-2018.proposal": " Submit Your Proposal",
-    "conference-2018.proposalDeadline": "Deadline for proposals: February 5",
+    "conference-2018.proposalDeadline": "February 5",.
```
Comment (Kenny2github) [inline src/views/conference/2018/index/l10n.json]: Please remove this period (`.`)

Comment (redmangospros) [reply src/views/conference/2018/index/l10n.json]: Okay!

Comment (RedMAngo22) [inline src/views/conference/2018/index/l10n.json]: @mewtaylor

**Comment by VutonDesign** [comment]
@RedMAngo22 You're the same person as @redmangospros...

**Comment by GreenBayRules** [comment]
Thats the way he rolls... XD


He does some thing on one account, complements himself on another account -_-

________________________________

**Comment by redmangospros** [comment]
THAT WAS A MISTAKE!

**Comment by GreenBayRules** [comment]
LOL

________________________________

File: src/views/conference/2018/index/l10n.json [hunk 2]
```diff
@@ -23,7 +23,7 @@
     "conference-2018.sessionItem4Desc": "Share what you've been doing in a short, lively presentation.",
 
     "conference-2018.proposal": " Submit Your Proposal",
-    "conference-2018.proposalDeadline": "Deadline for proposals: February 5",
+    "conference-2018.proposalDeadline": "February 5",
```
Comment (Kenny2github) [inline src/views/conference/2018/index/l10n.json]: I assume this was intentional?

Comment (redmangospros) [reply src/views/conference/2018/index/l10n.json]: Yeah. #1702

**Comment by redmangospros** [comment]
@mewtaylor @thisandagain  @Kenny2github  @me-win

**Comment by redmangospros** [comment]
@mewtaylor

**Comment by GreenBayRules** [comment]
I think this should be closed since most likely, someone will create a pull request that does all of the things that @me-win requested in his/her issue. This pull request also adds another line that is not requested in the issue: https://github.com/LLK/scratch-www/pull/1705/files#diff-647c54b0a5fdb8b9443d35965a7370e9R26

**Comment by redmangospros** [comment]
I will close this and re make it

**Pull request closed without merging**